examples/ucp_hello_world.c

```diff
@@ -400,13 +419,19 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
     ep_params.err_mode        = err_handling_opt.ucp_err_mode;
     ep_params.err_handler.cb  = failure_handler;
     ep_params.err_handler.arg = NULL;
-    ep_params.user_data       = &client_status;
+    ep_params.user_data       = &ep_status;
 
     status = ucp_ep_create(ucp_worker, &ep_params, &client_ep);
     /* If peer failure testing was requested, it could be possible that UCP EP
      * couldn't be created; in this case set `ret = 0` to report success */
     CHKERR_ACTION(status != UCS_OK, "ucp_ep_create\n",
-                  ret = (err_handling_opt.failure) ? 0 : -1; goto err);
+                  ret = (err_handling_opt.failure_mode != FAILURE_MODE_NONE) ? 0 : -1; goto err);
+
+    if (err_handling_opt.failure_mode == FAILURE_MODE_SEND) {
+        fprintf(stderr, "Emulating unexpected failure on server side: "
```

Comment:
Emulating unexpected failure on server side, client should detect error by keepalive mechanism
move this to line 399